proto/raft_serverpb.proto

```diff
@@ -60,6 +60,53 @@ message SnapshotChunk {
 
 message Done {}
 
+message TabletSnapshotFileMeta {
+    uint64 file_size = 1;
+    string file_name = 2;
+    // Some block data.
+    bytes head_chunk = 3;
```

Comment:
is it better to separate the checksum for quick compare?

Reply:
That will require parsing the SST file.